### DISCUSSION TITLE
docker on Windows instructions

### DISCUSSION THREAD

**Discussion by ericjgoebel** [mbround18/valheim-docker#801] [Help!] [answered]
Is there any windows specific instructions? I have a dedicated server running successfully on a Windows 10 Pro machine and would like to dockize it instead. I jumped through all the hoops to enable docker but noticed that the instructions provided for this repo are all linux centric. I know that it shouldn't matter what OS the host is and I could try running through the steps provided and just assume the Windows equivalent but if anyone has already done this it would be a great help as I'm not as familiar with docker.

**Comment by mbround18** [comment, marked as answer]
I build this on windows running docker, the docker compose should work no matter what operating system its on as long as it can run docker :)

**Reply by ericjgoebel** [reply]
Right, I was just noticing steps like this ...
**Steps**

> SSH into the server as a non-root user or su into non-root user. ([How to create a linux user](https://linuxize.com/post/how-to-create-users-in-linux-using-the-useradd-command/))
> Make sure they have access to docker usermod -aG docker ${USER}
> Create a new directory mkdir -p ./valheim

Are not really relevant to someone using a Windows UI and wondered if anyone had created windows specific steps. I assume in this case it would be like 

> Log in as an administrator
> Create a new directory/folder 'valheim'

etc

I know for me im a bit fuzzy on how to copy world files from my dedicated server to the mounted dir that the container will use but I'm hoping that someone already asked that and I can find that discussion somewhere. The https://github.com/mbround18/valheim-docker/blob/main/docs/tutorials/how-to-transfer-files.md seems like lots of extra steps for my situation

**Reply by mbround18** [reply]
on windows, a lot of thats handed to you and are extra steps. you should be able to compose up the compose out of the gate, transfering files between servers like that is still valid for window's use case

**Reply by ericjgoebel** [reply]
Oh, I may be making some poor assumptions on how that works then. I figured if the steps would be simpler if i had the world files on the host machine already.

**Reply by mbround18** [reply]
if you setup the volume mount you should have the files on the host machine
